### PR TITLE
The Event content type should use Smart Date fields, not datetime_range

### PR DESCRIPTION
**Pull request by mandclu**
As per the discussion in #64, convert the Event content type to use a Smart Date field, and set the default value to "next hour".

### CODE REVIEW

File: recipes/starshot_event_content_type/config/node.type.event.yml
Comment (phenaproxima) [inline recipes/starshot_event_content_type/config/node.type.event.yml:26]: Let's leave this as `null` for the moment, since this description feels a tad awkward to me. But we can open a follow-up PR to bikeshed it.

File: recipes/starshot_event_content_type/config/node.type.event.yml
Comment (phenaproxima) [inline recipes/starshot_event_content_type/config/node.type.event.yml:10]: I'd rather we remove all the `menu_ui` stuff. It seems unlikely that people will want to list _individual events_ in any of the menus.

File: recipes/starshot_event_content_type/recipe.yml
Comment (phenaproxima) [inline recipes/starshot_event_content_type/recipe.yml:12]: Let's revert this, events need not be exposed to menus.

File: recipes/starshot_event_content_type/config/pathauto.pattern.events.yml
Comment (phenaproxima) [inline recipes/starshot_event_content_type/config/pathauto.pattern.events.yml:12]: What does this resolve to? Will this be the start date or the end date?

File: recipes/starshot_event_content_type/config/node.type.event.yml
Comment (mandclu) [reply recipes/starshot_event_content_type/config/node.type.event.yml:10]: The change here is actually to exclude it from showing any available menus by default. Otherwise, new events have the option to be added to the main navigation menu.

File: recipes/starshot_event_content_type/config/pathauto.pattern.events.yml
Comment (mandclu) [reply recipes/starshot_event_content_type/config/pathauto.pattern.events.yml:12]: Yes, this will provide the start value formatted as a year. The tokens for Smart Date fields are a little different.

File: recipes/starshot_event_content_type/config/node.type.event.yml
Comment (mandclu) [reply recipes/starshot_event_content_type/config/node.type.event.yml]: Change made

File: recipes/starshot_event_content_type/config/node.type.event.yml
Comment (phenaproxima) [reply recipes/starshot_event_content_type/config/node.type.event.yml:10]: Gotcha - I think it's okay to exclude it anyway. I'd rather not incur the cost of installing an extra module (menu_ui) just to cover this third-party setting. :)

Comment (mandclu) [reply recipes/starshot_event_content_type/config/node.type.event.yml]: Removed.

**Comment by phenaproxima** [comment]
Just FYI -- I cannot merge this because it's not installing. This is the error we're getting: https://github.com/phenaproxima/starshot-prototype/actions/runs/9071250306/job/24967409796?pr=69#step:5:791

Probably some misconfiguration in the fields.

**Comment by mandclu** [comment]
OK I'll try to have a look tomorrow

**Comment by phenaproxima** [comment]
Looks like there's a bug in the module here, @mandclu...bad enough that it prevents installation. Can you address that? I'll need to at least manually test this before I can merge it.

**Comment by mandclu** [comment]
I added a commit and then tested the result on a fresh install. The updated recipe worked without issue for me.

**Review by phenaproxima — APPROVED** [review]
No problems with the code.

**Comment by phenaproxima** [comment]
I think this needs to be rebased against `main`.

**Pull request merged**
